nvflare/job_config/script_runner.py

```diff
@@ -12,12 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Type
+from nvflare.client.config import TransferType
 
-from nvflare.app_common.executors.client_api_launcher_executor import ClientAPILauncherExecutor
-from nvflare.app_common.executors.in_process_client_api_executor import InProcessClientAPIExecutor
-from nvflare.client.config import ExchangeFormat, TransferType
-from nvflare.fuel.utils.import_utils import optional_import
+from .base_script_runner import BaseScriptRunner
```

Comment:
Also import FrameworkType from BaseScriptRunner, and remove it from this file.

Reply:
merge the two files as suggested.